camp_1/openzeppelin/access/ownable/library.cairo

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I don't think we should add OZ's dependencies to our repo, instead let's provide installation instructions if needed